modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -522,6 +534,33 @@ else if (ctEntry.getChangeType() !=
 			}
 		}
 
+		String workflowView = null;
+
+		Map<String, String> workflowData = new LinkedHashMap<>();
+
+		if (rightModel instanceof WorkflowedModel) {
+			long groupId = 0;
+
+			if (rightModel instanceof GroupedModel) {
+				GroupedModel groupedModel = (GroupedModel)rightModel;
+
+				groupId = groupedModel.getGroupId();
+			}
+
+			if ((groupId != 0) &&
+				_ctDisplayRendererRegistry.isWorkflowEnabled(
+					ctEntry.getCompanyId(), groupId,
```

Comment:
Shouldn't we check for `changeType != CTConstants.CT_CHANGE_TYPE_DELETION` here as well to be consistent with the logic you have in `ViewChangesDisplayContext`?